src/snapshot.zig

```diff
@@ -676,6 +688,23 @@ fn processSnapshotFile(gpa: Allocator, snapshot_path: []const u8, maybe_fuzz_cor
         try writer.writeAll("\n");
     }
 
+    // Write TYPES SECTION
+    {
+        var solved = std.ArrayList(u8).init(gpa);
+        defer solved.deinit();
+
+        try can_ir.toSexprTypesStr(solved.writer().any(), maybe_expr_idx, content.source);
+
+        // Uncomment to print entire types store, helpful for debugging
+        // try solved.writer().any().writeAll("\n");
+        // try types.SExprWriter.allVarsToSExprStr(solved.writer().any(), gpa, &module_env);
```

Comment:
We've got flags like `--verbose` on the snapshot tool. We might add one like `--debug-type-store`?